lib/app/components/list_item/variants/list_item_user.dart

```diff
@@ -4,9 +4,10 @@ part of '../list_item.dart';
 
 class _ListItemUser extends ListItem {
   _ListItemUser({
-    required String pubkey,
```

Comment:
Maybe it makes sense to use `ListItem` directly if u don't need to pass the pubkey?
```
ListItem({leading: Avatar(...), ...});
```